test/request.js

```diff
@@ -1398,7 +1398,37 @@ describe('request', function() {
       }).end()
     })
 
-  })
+    it('should allow for block-wise transfer when using multicast', function (done) {
+      var payload = Buffer.alloc(1536)
+        , counter = 0  
+      
+      server = coap.createServer((req, res) => {
+        expect(req.url).to.eql("/hello")
+        res.end(payload)
+      })
+      server.listen(sock)
+      
+      var server2 = coap.createServer((req, res) => {
+        expect(req.url).to.eql("/hello")
+        res.end(payload)
+      })
+      server2.listen(sock)
+
+      var _req = request({
+        host: MULTICAST_ADDR,
+        port: port2,
+        pathname: "/hello",
```

Comment:
I extended now the existing test to check for the correct URI path. I think this should work now.